recipe/scripts/activate.sh

```diff
@@ -0,0 +1,2 @@
+export JULIA_DEPOT_PATH_BACKUP=${JULIA_DEPOT_PATH:-}
+export JULIA_DEPOT_PATH="$CONDA_PREFIX/share/julia/site:$JULIA_DEPOT_PATH"
```

Comment:
```suggestion
export JULIA_DEPOT_PATH_BACKUP=${JULIA_DEPOT_PATH:-}
export JULIA_PROJECT_BACKUP=${JULIA_PROJECT:-}
export JULIA_DEPOT_PATH="$CONDA_PREFIX/share/julia/site:$JULIA_DEPOT_PATH"
export JULIA_PROJECT="$CONDA_PREFIX/share/julia/site/environments/$CONDA_DEFAULT_ENV"
```

Create a new shared environment located in the site depot for the environment rather than continuing to use the default one in `~/.julia/environments/v#.#`.

The alternative is to just set the depot to only the site depot without prefixing it to either a pre-existing `JULIA_DEPOT_PATH` or the default one via a trailing `:`.
```
export JULIA_DEPOT_PATH="$CONDA_PREFIX/share/julia/site"
```

Reply:
I think keeping the depots stacked and creating a Julia environment corresponding to the conda environment creates the best situation. It allows the conda environment to use the contents in `~/.julia`, which means less duplication of files such as the registry.